repository/src/main/java/org/apache/atlas/repository/store/graph/v1/DeleteHandlerV1.java

```diff
@@ -590,6 +590,26 @@ public boolean isRelationshipEdge(AtlasEdge edge) {
         return ret;
     }
 
+    public List<AtlasVertex> removeTagPropagation(AtlasClassification classification, List<AtlasEdge> propagatedEdges) throws AtlasBaseException {
+        List<AtlasVertex> ret = new ArrayList<>();
+
+        for (AtlasEdge propagatedEdge : propagatedEdges) {
+            AtlasPerfMetrics.MetricRecorder metric = RequestContext.get().startMetricRecord("removeTagPropagationEdge");
```

Comment:
metric log for "removeTagPropagationEdge" is duplicated, there is another method with this same title for metric, rename to keep it unique